apps/src/templates/PaneHeader.jsx

```diff
@@ -192,18 +196,26 @@ export const PaneButton = Radium(function(props) {
     }
   }
 
+  function onClickIfEnabled() {
+    if (props.isDisabled) {
+      return;
+    }
+    props.onClick();
+  }
+
   return (
-    <div
+    <button
+      type="button"
       id={props.id}
-      style={divStyle}
-      onClick={props.isDisabled ? () => {} : props.onClick}
+      style={buttonStyle}
+      onClick={onClickIfEnabled}
```

Comment:
now that this element is a button, we should be able to use the `disabled` attribute and remove the `onClick` logic that checks whether or not the element is disabled